CMakeLists.txt

```diff
@@ -63,6 +63,8 @@ endif()
 add_subdirectory(thirdparty)
 add_subdirectory(src)
 add_subdirectory(samples)
+add_subdirectory(tools/continuous_batching/accuracy)
+add_subdirectory(tools/continuous_batching/benchmark)
```

Comment:
suggest creation of tools/CmakeLists.txt with these 2 clauses.

Reply:
Created